C++/source/graph/Dijkstra.cpp

```diff
@@ -0,0 +1,49 @@
+// Author : Abdelrahman Hammam
+//Github handle : AbdoHammam
```

Comment:
Please use the description comment format specified in the [contribution guidelines](https://github.com/faheel/Algos/blob/reorganise/.github/CONTRIBUTING.md).

You can add your name at the end of it, like so:

```
    Author
    ------
    Abdelrahman Hammam (@AbdoHammam)
```